features/step_definitions/shared/xcode.rb

```diff
@@ -107,3 +107,17 @@
   assert_equal phase_name, build_phase.name
 end
 
+Then(/^in project (.+) the target (.+) should contain the build phase (.+)$/) do |project_name, target_name, phase_name|
```

Comment:
We can delete this one. We are already checking whether the final product has got the clip copied, which is what we are expecting in the end.